pints/_mcmc/__init__.py

```diff
@@ -399,6 +446,8 @@ def chains(self):
         """
```

Comment:
If was expecting something near line 398 "self._needs_sensitivities = ..." that checks that the transform provides them too. I imagine this is not always the case?

Reply:
Hmm?

Reply:
The controller checks if sensitivities are available when the method requires them.
This means the transformed objects should also have sensitivities available.
Didn't read the code in detail, so this might be automatic? But otherwise we'd have to update that check a bit to test that the provided transformation is able to do sensitivities

Reply:
Hmm we don't check if sensitivities are available or not (by far I can see/search!). It only asks if the method requires sensitivity
```
        # Check if sensitivities are required
        self._needs_sensitivities = self._samplers[0].needs_sensitivities()
```
Then it went straight to ask for it
```
        # Choose method to evaluate                                             
        f = self._log_pdf                                                       
        if self._needs_sensitivities:                                           
            f = f.evaluateS1
```
So the `TransformLogPDF` that has `evaluateS1` should automatically deal with this.

Reply:
I will mark this as resolved too. Repoen this if you think I've got this wrong.